src/nouveau/src/nouveau_fabric_search.erl

```diff
@@ -215,11 +222,43 @@ convert_item(Item) ->
             maps:get(<<"value">>, Item)
     end.
 
-merge_facets(FacetsA, null, _Limit) ->
+top_n_facets(SearchResults, TopN) ->
+    After = SearchResults#{
```

Comment:
No need for `After` if we're just returning the result anyway?

Reply:
leftover from some logging I removed.